### ISSUE TITLE
Damping of meltwater perturbation by sea ice change

### ISSUE THREAD

**Issue by adele157** [adele157/IAF-meltwater#18]
Matt proposed the following sea ice mechanism that may be damping the impact of our meltwater perturbation (in addition to the [30% damping by the salinity restoring](https://github.com/adele157/IAF-meltwater/issues/2#issuecomment-1247471580)):

- There is more sea ice production over the shelf in the meltwater case.
- This sea ice melts further north (not over the shelf).
- Therefore enhanced sea ice production removes freshwater from the shelf, offsetting the additional meltwater we are adding there.

This is easy to check. Plot a time series of the change in freshwater flux from sea ice (i.e. melt and salt flux from ice terms) and compare to the change in runoff.

**Comment by adele157** [comment]
Indeed, this also contributes a 30% reduction to our meltwater addition:
<img width="600" alt="Screen Shot 2022-09-15 at 12 25 17 pm" src="https://user-images.githubusercontent.com/8506963/190298729-881c13ac-b4dc-40fd-9fdb-92db6707cf00.png">

So our added meltwater is being reduced by 60% due to sea ice freshwater changes and salinity restoring.

This sea ice mechanism is a real effect right though? As in, it would also happen in the real world?

**Comment by taimoorsohail** [comment]
What I gathered from the discussion was that this effect is most pronounced when the meltwater addition occurs only at the surface. In the real world, the meltwater is also added at depth which means it wouldn't have such immediate feedback on sea ice formation at the surface as it entrains with intermediate waters on the shelf.

**Comment by adele157** [comment]
Ah, yes, good point! This is exactly what Pedro has been finding in his
basal melt parameterisation runs. Changing runoff at the surface has a huge
impact on sea ice growth, but altering basal melt at depth has nearly no
impact on sea ice.
